src/app/pages/Languages.svelte

```diff
@@ -124,6 +133,11 @@
 <main>
   <div class="languages">
     <div class="languages__container">
+      <Link to="/">
+        <span class="button button--grey button--outline mt-xxl">
+          <div class="icon"><IoIosArrowBack /></div>Go back
+        </span>
+      </Link>
```

Comment:
This markup should be simplified.

```suggestion
      <a href="/" class="button button--grey button--outline mt-xxl">
          <IoIosArrowBack /> Go back
      </a>
```

For now, do not use the `<Link>` element, because `svelte-check`/TypeScript is having a coniption about it, and this particular usage of the `<Link>` component does not use routes; it is a link to "/" which can be done with an `<a>` element.

As for the size of the icon, I would customize its `viewport` to be the desired size as well as tweak its vertical alignment. The `<IoIosArrowBack />`becomes a `<svg>` in the DOM, so you can target that in selectors!